charts/chainlink-cluster/README.md

```diff
@@ -136,15 +136,23 @@ helm uninstall cl-cluster
 ```
 
 # Grafana dashboard
-We are using [Grabana]() lib to create dashboards programmatically
+We are using [Grabana](https://github.com/K-Phoen/grabana) lib to create dashboards programmatically
+
+You can select `PANELS_INCLUDED`, options are `core`, `wasp`, comma separated
```

Comment:
for now only core (wasp in already included by just specifying INFRA_PLATFORM to kubernetes)